src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1914,7 +1914,17 @@ private AssemblySymbol GetForwardedToAssembly(string fullName, int arity, out bo
 
         internal static void CheckFeatureAvailability(Location location, MessageID feature, DiagnosticBag diagnostics)
         {
-            LanguageVersion availableVersion = ((CSharpParseOptions)location.SourceTree.Options).LanguageVersion;
+            var options = (CSharpParseOptions)location.SourceTree.Options;
```

Comment:
Not sure. I do know that this method is called in a few places in a place for non-syntax-features, e.g. https://github.com/khyperia/roslyn/blob/dff492c59ac5a2da70d89b32b5f9d12799bf8fd0/src/Compilers/CSharp/Portable/Binder/SwitchBinder.cs#L344

Also, CSharpCompilationOptions.Features contains the same items as CSharpParseOptions.Features, so if a CompilationOptions is in scope somewhere, that might be better to grab than ParseOptions?